drivers/wifi/nrf700x/zephyr/src/zephyr_coex.c

```diff
@@ -47,15 +47,15 @@ static const struct gpio_dt_spec btrf_switch_spec =
 /* PTA registers configuration of Coexistence Hardware */
 /* Separate antenna configuration, WLAN in 2.4GHz*/
 const uint16_t config_buffer_SEA[] = {
-	0x19, 0xF6, 0x8, 0x62, 0xF5,
-	0xF5, 0x19, 0x19, 0x74, 0x74,
-	0x8, 0x1E2, 0xD5, 0xD5, 0x1F6,
-	0x1F6, 0x61, 0x61, 0x1E2, 0x8,
-	0x1F6, 0x1F6, 0x19, 0x19, 0x1E2,
-	0x8, 0xF5, 0xF5, 0xD5, 0xD5,
-	0x8, 0x1E2, 0x51, 0x51, 0x74,
-	0x74, 0xF6, 0x19, 0x62, 0x19,
-	0xF6, 0x8, 0x62, 0x8, 0x1A
+	0x0019, 0x00F6, 0x0008, 0x0062, 0x00F5,
```

Comment:
can you please do this change in a separate commit?

Reply:
Done.